src/FluxCalculator.jl

```diff
@@ -91,4 +100,296 @@ function calculate_surface_air_density(atmos_sim::Interfacer.AtmosModelSimulatio
     error("this function is required to be dispatched on" * Interfacer.name(atmos_sim) * ", but no method defined")
 end
 
+"""
+    partitioned_turbulent_fluxes!(model_sims::NamedTuple, fields::NamedTuple, boundary_space::Spaces.AbstractSpace, surface_scheme, thermo_params::TD.Parameters.ThermodynamicsParameters)
+
+The current setup calculates the aerodynamic fluxes in the coupler (assuming no regridding is needed)
+using adapter function `get_surface_fluxes_point!`, which calls `SurfaceFluxes.jl`. The coupler saves
+the area-weighted sums of the fluxes.
+
+Args:
+- `model_sims`: [NamedTuple] containing `ComponentModelSimulation`s.
+- `fields`: [NamedTuple] containing coupler fields.
+- `boundary_space`: [Spaces.AbstractSpace] the space of the coupler surface.
+- `surface_scheme`: [AbstractSurfaceFluxScheme] the surface flux scheme.
+- `thermo_params`: [TD.Parameters.ThermodynamicsParameters] the thermodynamic parameters.
+
+TODO:
+- generalize interface for regridding and take land state out of atmos's integrator.p
+- add flux accumulation
+- add flux bounds
```

Comment:
Will these be tackled in an upcoming SDI?

Reply:
It's not something planned immediately, probably after the release.